src/components/Sidebar/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import './style.css'
+import { Link, NavLink } from 'react-router-dom';
+import Navbar from 'react-bootstrap/Navbar';
+import Nav from 'react-bootstrap/Nav';
+import { Call, LogoGithub, LogoLinkedin, Mail, LogoBehance, LogoInstagram } from 'react-ionicons'
+
+const Sidebar = () => {
+    return (
+        <div className="sidebar">
+            <Link to="/" className="to-home">Ayushi Sood</Link>
+            <div className="contact-container">
+                <Mail
+                    height="25px"
+                    width="25px"
+                />
+                <div className="email">sood.ayushi30@gmail.com</div>
+            </div>
+            <div className="contact-container">
+                <Call
+                    height="25px"
+                    width="25px"
+                />
+                <div className="mobile">+91-9466460493</div>
```

Comment:
you'll make your number PUBLIC ?

Reply:
yeah. decided that phone number is not needed. Hence, removed.